src/node/internal/crypto_hash.ts

```diff
@@ -173,4 +183,76 @@ Hash.prototype.digest = function(this: Hash, outputEncoding?: string): Buffer |
   }
 }
 
-export {Hash};
+///////////////////////////
+
+interface Hmac extends Transform {
+  [kHandle]: cryptoImpl.HmacHandle;
+  [kState]: _kState;
+}
+
+export function createHmac(hmac: string, key: ArrayLike | KeyObject | CryptoKey,
+                           options?: TransformOptions): Hmac {
+  return new Hmac(hmac, key, options);
+}
+
+let Hmac = function(this: Hmac, hmac: string, key: ArrayLike | KeyObject | cryptoImpl.CryptoKey,
+                    options?: TransformOptions): Hmac {
+  if (!(this instanceof Hmac))
+    return new Hmac(hmac, key, options);
```

Comment:
Nit:
```suggestion
  if (!(this instanceof Hmac)) {
    return new Hmac(hmac, key, options);
  }
```

Reply:
Same below. Where it actually gets a bit tough to see which statements are in the if branch (hence why I think it's best to be explicit).